api/api.proto

```diff
@@ -803,6 +803,206 @@ service OpenStorageVolume {
     }
 }
 
+service OpenStorageCredentials {
+
+  // Provide credentials to OpenStorage and if valid,
+  // it will return an identifier to the credentials
+  rpc ProvideForAWS(ProvideCredentialsForAWSRequest)
```

Comment:
Either use all "ProvideCredentialsXXX" or "CreateCredentialsXXX". Same for the REST urls. either `credentials/provide` or `credentials/create`.

My vote is not to use Create since we are not creating credentials. Either the verb could be Provide, or Submit, or Register, or something like that, but they all need to be consistent.

Reply:
It should be Create since it is creating the Credentials object and also keeps it inline with the other operations (volume, snapshot, etc)

Reply:
That's fine. Then all should be Create.